lib/starknet_explorer/block_fetcher.ex

```diff
@@ -1,56 +1,37 @@
 defmodule StarknetExplorer.BlockFetcher do
-  use GenServer
+  use GenServer, restart: :temporary
   require Logger
   alias StarknetExplorer.{Rpc, BlockFetcher, Block}
-  defstruct [:block_height, :latest_block_fetched]
-  @fetch_interval 10 * 60 * 1000
+  defstruct [:latest_block_number_fetched]
+  @fetch_latest_interval 100
   def start_link(args) do
     GenServer.start_link(__MODULE__, args)
   end
 
   @impl true
   def init(_opts) do
     state = %BlockFetcher{
-      # The actual chain block-height
-      block_height: fetch_block_height(),
-      # Highest block number stored on the DB
-      latest_block_fetched: StarknetExplorer.Block.highest_fetched_block_number()
+      latest_block_number_fetched: block_height()
     }
 
-    Process.send_after(self(), :fetch_and_store, @fetch_interval)
+    Process.send_after(self(), :fetch_latest, @fetch_latest_interval)
+    Logger.info("Starting explorer with fetcher enabled!")
```

Comment:
you can add "Starting explorer with block fetcher enabled!"

Reply:
So someone that starts the explorer for the first time has more context of what that log line means.